### DISCUSSION TITLE
[27 giugno 2023] Esercizio 4

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/psd2223#255] [Esami]
$\Large a)$ Convertire il numero $\Large X=29,625$ nel formato IEEE half-precision.

$\Large b)$ Preso il numero esadecimale $\Large CBD0$, convertirlo in binario ed interpretare questa sequenza come un numero $\Large Y$ razionale in formato IEEE half-precision.

$\Large c)$ Calcolare sia $\Large X+Y$ che $\Large X-Y$ e scrivere I risultati in formato IEEE half-precision.

> [!NOTE]
> Solo per la traccia del canale M-Z nel punto $\Large "b)"$ $\Large CBD0$ va convertito in una sequenza binaria di 16 bit

**Comment by M1SC0** [comment]
![image](https://github.com/user-attachments/assets/38914b6b-e29f-413b-8618-319bc01d43f9)
